plugins/Denizen/scripts/cooking/allergies.dsc

```diff
@@ -0,0 +1,78 @@
+allergies:
+    type: world
+    debug: false
+    events:
+        on player dies:
+
+            - define allergy <script[allergy_info].data_key[valid_allergies].keys.random>
+            - define effect <script[allergy_info].data_key[valid_effects].keys.random>
+            - flag player allergic:<[allergy]>
+            - narrate <[allergy]>
+            - flag player allergy_effect:<[effect]>
+            - stop
+            - flag player allergic:!
+            - flag player allergy_effect:!
```

Comment:
stray debugging tool?

Reply:
> stray debugging tool?

FYI, for drafts they typically are being tested/worked on and the debugging stuff should be addressed when the PR is ready to be pulled. @creaturesofhabit